server/externs/mongodb.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Node.js mongodb externs.
+ */
+
+/**
+ * @constructor
+ * @return {!mongodb}
+ */
+function mongodb() {}
+
+/**
+ * @constructor
+ * @return {!cursor}
+ */
+function cursor() {}
+
+/** @const */
+mongodb.MongoClient;
+
+/**
+ * @param {string} url
+ * @param {function()} callback
+ */
+mongodb.MongoClient.connect = function(url, callback) {};
+
+/**
+ * @constructor
+ * @return {!mongodb.Db}
+ */
+mongodb.Db = function() {};
+
+/**
+ * @param {string} dbName Name of database.
+ */
+mongodb.Db.prototype.collection = function(dbName) {};
+
+/**
+ * @param {Object=} query
+ * @param {Object=} projection
+ * @return {!cursor}
+ */
+mongodb.Db.prototype.collection.find = function(query, projection) {};
+
+/**
+ * @param {Object} document
```

Comment:
Objects need to be non-null explicitly. 
```js
{!Object}
```
Preferably you may want to expand some of the objects defined here to detailedly list their attributes, at least for those used.